Mathlib/Data/PFunctor/Univariate/Basic.lean

```diff
@@ -13,9 +13,7 @@ polynomial functor.  (For the M-type construction, see
 pfunctor/M.lean.)
```

Comment:
Only in scope if you want it to be, but it would be great to fix this link to a real `Mathlib/../*.lean` path